client.js

```diff
@@ -224,9 +261,20 @@ class Client {
    *   { dataConditions: { "Text": "Invalid or missing feature license detected." } }
    *
    * @param {Object} query - A simple plain object representing the EventQuery.
-   * @returns {Promise<Array>} Resolves with an array of events (each event 
-   *   includes a `codeDescription` which provides a textual representation 
-   *   of the event code).
+   * @returns {Promise<Array>} Resolves with an array of event objects.
+   *
+   * Each event object typically includes the following fields:
+   *  - `sender` (string): The event sender.
+   *  - `data` (object): An object containing event-specific details:
+   *       - `Text` (string): The event text message.
+   *       - `Level` (string): The event level (e.g., "ERROR").
+   *       - `Description` (string): A detailed description of the event.
+   *       - `Group` (string): A group identifier for the event.
+   *  - `timestampSec` (number): The timestamp (in seconds) when the event occurred.
+   *  - `id` (string): A unique identifier for the event.
+   *  - `code` (number): The raw event code returned by the server.
+   *  - `status` (number): The status code associated with the event.
+   *  - `logstampSec` (number): The log timestamp (in seconds) when the event was logged.
    */
   requestEvents(query) {
```

Comment:
Are the `CountEventsRequest` and `CountEventsResponse` in database.proto also not implemented yet?

Basically, should add a `countEvents(query)` method. The `query` is in same format as in `requestEvents(query)` but the `countEvents` just returns the number of events. This is useful to just display how many events there are in the database without needing to actually fetch all the data.